plugins/workspace-minimap/src/focus_mode.ts

```diff
@@ -19,12 +20,12 @@ const BlockEvents = new Set([
   Blockly.Events.BLOCK_DRAG,
   Blockly.Events.BLOCK_MOVE]);
 
+const BorderRadius = 6;
```

Comment:
Nit: CONSTANT_CASE for constants.

Reply:
I get this eslit error:

Variable name `BORDER_RADIUS` must match one of the following formats: camelCase, PascalCase eslint@typescript-eslint/naming-convention 
const BORDER_RADIUS: 6

Reply:
Ha ok well I'm not surprised our eslint doesn't match the style guide :P I would change to camelCase then since PascalCase is used for classes.